frontend/src/scenes/insights/InsightTooltip/index.tsx

```diff
@@ -5,37 +5,44 @@ import { IntervalType } from '~/types'
 import './InsightTooltip.scss'
 
 interface BodyLine {
-    id: string
+    id?: string | number
     component: JSX.Element
 }
 
 interface InsightTooltipProps {
     chartType: string
-    altTitle?: string // Alternate string to display as title (in case date reference is not available, e.g. when comparing previous)
+    altTitle?: string | JSX.Element | null // Alternate string to display as title (in case date reference is not available or not desired)
     referenceDate?: string
-    interval: IntervalType
-    bodyLines: BodyLine[]
+    interval?: IntervalType
+    bodyLines?: BodyLine[]
     inspectUsersLabel?: boolean
+    children?: React.ReactNode
 }
 
 export function InsightTooltip({
     chartType,
     altTitle,
     referenceDate,
     interval,
-    bodyLines,
+    bodyLines = [],
     inspectUsersLabel,
+    children,
```

Comment:
`bodyLines` is in here for its similarity to the charting library's built-in tooltips, but `children` is easier to use in other React components